comps/comps-foreman-plugins-fedora24.xml

```diff
@@ -65,7 +65,6 @@
       <packagereq type="default">rubygem-smart_proxy_openscap</packagereq>
       <packagereq type="default">rubygem-smart_proxy_pulp</packagereq>
       <packagereq type="default">rubygem-smart_proxy_remote_execution_ssh</packagereq>
-      <packagereq type="default">rubygem-smart_proxy_remote_execution_ssh_core</packagereq>
```

Comment:
This is from #1348, if you could leave it here for now it'll make the merge much easier (I'll mess something up changing five packages at once.)